integrations/discord-bot/voxguard/dashboard/static/style.css

```
@@ -0,0 +1,368 @@
+.btn-primary:hover { filter: brightness(1.08); }
+.btn-primary:active { transform: translateY(1px); }
+
+.btn-ghost {
+  background: transparent; color: var(--text-dim);
+  border: 1px solid var(--border-lit); border-radius: var(--radius-sm);
+  padding: 8px 12px; cursor: pointer; font-family: inherit; font-size: 13px;
+  display: inline-flex; align-items: center; gap: 6px;
+  transition: background .15s, color .15s, border-color .15s;
+}
+.btn-ghost:hover { background: var(--bg-hover); color: var(--text); border-color: var(--brand); }
+.btn-ghost svg { width: 15px; height: 15px; fill: currentColor; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)